packages/react-next/src/components/Checkbox/useCheckbox.tsx

```diff
@@ -0,0 +1,75 @@
+import * as React from 'react';
+import { ICheckboxProps } from './Checkbox.types';
+import { ComposePreparedOptions } from '@fluentui/react-compose';
+import { mergeProps } from './temp';
+import { useControllableValue, useId } from '@uifabric/react-hooks';
+
+export const useCheckbox = (props: ICheckboxProps, options: ComposePreparedOptions) => {
+  const {
+    disabled,
+    inputProps,
+    name,
+    ariaLabel,
```

Comment:
as discussed in https://hackmd.io/k2eJKI_JRs2Gew9dbrz00A?view, please use dashed properties (`aria-label`) instead of camel case (`ariaLabel`) for aria-* props